lnencrypt/tor_test.go

```diff
@@ -0,0 +1,60 @@
+package lnencrypt
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/lntest/channels"
+	"github.com/lightningnetwork/lnd/lntest/mock"
+	"github.com/stretchr/testify/require"
+)
+
+// TestTorPrivateKey tests writing and reading a private key to disk.
+func TestTorPrivateKey(t *testing.T) {
```

Comment:
This test references `Read/WriteTorPrivateKey` which haven't been introduced yet. We require each individual commit in the codebase to compile.